### ISSUE TITLE
[BUG] Can't get info

### ISSUE THREAD

**Issue by ruehj** [sleeyax/aternos-discord-bot#46]
**Describe the bug**
The bot can't communicate through aternos and it always says "Can't get info." I tried it unto other server but it worked.

**Expected behavior**
The server should open if I use the slash commands, but it always saying the same line.

**Comment by sleeyax** [comment]
So it works with one minecraft server but it doesn't with another? Please provide more details to help me understand your issue better.

**Comment by ruehj** [comment]
I mean, it works on another discord server, but when I try it on the server
it supposed to be, it shows this message.

**Comment by sleeyax** [comment]
That's odd, have you tried kicking the bot and reconfiguring it again?

**Comment by ruehj** [comment]
I did, but let me try again some time

**Comment by sleeyax** [comment]
Is this other discord server linked to the same aternos account?

**Comment by ruehj** [comment]
The other discord server that I've tried the bot was using the same aternos
account

**Comment by sleeyax** [comment]
>  let me try again some time

Any news @ruehj ?

**Comment by sleeyax** [comment]
Closing this due to no response from OP about the current state of this issue.
